src/main/java/duke/misc/Ui.java

```diff
@@ -0,0 +1,16 @@
+package duke.misc;
+
+/**
+ * Ui class to encapsulate display messages and handle input/output of duke.
+ */
+public class Ui {
+    /** Welcome message. */
+    public static final String WELCOME_MSG = "Hi I am Annie!\nHow can I assist you?";
```

Comment:
I really like the way of organizing all the messages in a different class. Maybe you can also make use of Java enum here something like this. This is the way I have organized my messages. 

![Screenshot 2021-09-05 at 11 04 57 PM](https://user-images.githubusercontent.com/55918675/132131613-472c8bcb-33fa-4469-a85d-a877bb40eadc.png)